src/PaprController.sol

```diff
@@ -368,7 +368,7 @@ contract PaprController is
         onlyOwner
     {
         for (uint256 i = 0; i < collateralConfigs.length;) {
-            if (collateralConfigs[i].collateral == address(0)) revert IPaprController.InvalidCollateral();
+            if (address(collateralConfigs[i].collateral) == address(0)) revert IPaprController.InvalidCollateral();
 
             isAllowed[collateralConfigs[i].collateral] = collateralConfigs[i].allowed;
             emit AllowCollateral(collateralConfigs[i].collateral, collateralConfigs[i].allowed);
```

Comment:
not related, but `AllowCollateral` event name is a little misleading here, since it would be emitted on a dis allow, maybe `AllowedCollateralChange` or something

Reply:
Yeah I was kind of modeling on the erc721 ApprovalForAll which has an approved Boolean as an arg